lib/src/maintenance.dart

```diff
@@ -274,6 +276,29 @@ Future<Maintenance> detectMaintenance(String pkgDir, String pkgName,
         penalty: new Penalty(fraction: 200)));
   }
 
+  // Checking the length of description.
+  final description = pubspec.description?.trim();
+  if (description == null || description.isEmpty) {
+    maintenanceSuggestions.add(new Suggestion.warning(
```

Comment:
Pretty sure `pub` won't let you upload a package without a description – but not bad to have, I guess.